packages/react-dom/src/__tests__/ReactDOMServerIntegrationPure-test.internal.js

```diff
@@ -0,0 +1,112 @@
+/**
```

Comment:
There's a lot of setup cost for initializing these test modules. Can we put this together with some other tests since there's only a few of test cases here?